src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -97,7 +97,7 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
 
     public async componentDidMount() {
         const projectId = this.props.match.params["projectId"];
-        if (this.props.project) {
+        if (this.state.project) {
```

Comment:
We shouldn't need `state` project here.

Reply:
We should deprecate `project` from the component state since we are not editing without saving the project.

Reply:
Completely agree.  If we don't want to alter the project metadata adding the "all" assets list then definitely it should be better to completely remove project from the state as it will return to be a read only prop.